acceptance/tests/language/sensitive_data_type.rb

```diff
@@ -75,13 +84,20 @@
           run_assertions(assertion_code, result)
         end
 
-        if agent['platform'] =~ /fedora|centos|el|redhat|scientific|ubuntu|debian|cumulus/
```

Comment:
Do you want me to have the platforms listed in the `if agent ['platform']...` line and in the case statement then?  Or do you think it would be enough just to put a `logger.warn "we're not testing logs...` message in the else block for the case statement?